effektif-workflow-api/src/main/java/com/effektif/workflow/api/condition/And.java

```diff
@@ -58,18 +58,16 @@ public And condition(Condition condition) {
 
   @Override
   public void readBpmn(BpmnReader r) {
-    for (XmlElement andElement : r.readElementsEffektif(getClass())) {
-      r.startElement(andElement);
-      conditions = r.readConditions();
-      r.endElement();
-    }
+    conditions = r.readConditions();
```

Comment:
This looks like it was broken before to start/end reading the 'and' element. Was that the bug?